backend/app/main.py

```
@@ -0,0 +1,30 @@
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+# New endpoint to create a user
+@app.post("/users/")
+def create_user(user: User):
```

Ruff (0.8.2)

Undefined name `User`

(F821)